recipes/python-wand/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.5.5" %}
+{% set hash_value = "c97029751f595d96ae0042aec0e26ff114e403e060ae2481124abbcca0c65ce2" %}
+
+package:
+  name: python-wand
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/emcconville/wand/releases/download/0.5.5/Wand-0.5.5.tar.gz
```

Comment:
Use `version` jinja variable instead of `0.5.5`

Reply:
Done